### PR TITLE
Fix spelling errors

### PR DESCRIPTION
**Pull request by anisalibegic**
## Description

With the help of typo checker, I hand checked every file that had some typos in the docs or comments and fixed them.

## Tests

No tests for typos correction.

## Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I am willing to follow-up on review comments in a timely manner.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] All existing and new tests are passing.
- [ ] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.

## Breaking Change

- [ ] Yes, this is a breaking change.
- [x] No, no existing tests failed, so this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

**Comment by timsneath** [comment]
Great job on spotting a test that seems to be broken because of a spelling error.

File: packages/flutter/lib/src/widgets/editable_text.dart [hunk 1]
```diff
@@ -952,7 +952,7 @@ class EditableText extends StatefulWidget {
   ///               context: context,
   ///               builder: (BuildContext context) {
   ///                 return AlertDialog(
-  ///                   title: const Text('Thats correct!'),
+  ///                   title: const Text('That\'s correct!'),
```
Comment (xster) [inline packages/flutter/lib/src/widgets/editable_text.dart:955]: Just use double quotes. Makes it easier to read.

File: packages/flutter/test/painting/image_stream_test.dart [hunk 2]
```diff
@@ -418,7 +418,7 @@ void main() {
     await tester.pump(const Duration(milliseconds: 200));
   });
 
-  testWidgets('animation doesnt repeat more than specified', (WidgetTester tester) async {
+  testWidgets('animation doesn\'t repeat more than specified', (WidgetTester tester) async {
```
Comment (xster) [inline packages/flutter/test/painting/image_stream_test.dart:421]: same. Just use double quotes.

**Review by xster — COMMENTED** [review]
LGTM. Thanks for the clean-up. cc @jacob314 for the test fix revealed by the typo.

**Comment by anisalibegic** [comment]
@xster I pushed a [new version](https://github.com/flutter/flutter/pull/69704/commits/6a00bf5b9b37625d547554be47d0a1ff5154f56e) with the fixes you mentioned!

**Review by jacob314 — COMMENTED** [review]
lgtm for typo in inspector test.

**Comment by xster** [comment]
@jacob314 I meant to ask that now the inspector test fails with the correct spelling. There was likely an incorrect assertion in the test this whole time. Can you help @anisalibegic determine the correct test change since it might not be trivial?

**Comment by anisalibegic** [comment]
@xster @jacob314 So, what's up guys?

**Comment by xster** [comment]
@jacob314 can you help with the test?

**Pull request closed without merging**